client/src/pages/Invitation/Invitation.test.tsx

```diff
@@ -11,13 +11,29 @@ import {
   CurrentUserDocument,
   GetInvitationInfoDocument,
   AcceptInvitationDocument,
+  ListCurrentUserRoomsDocument,
 } from "graphql/generated/graphql";
 
 import Invitation from "./Invitation";
 import NotificationsWithPortal from "../../NotificationWithPortal";
 
 const FAKE_TOKEN = "1ace01d808072fc4905f1796203f8ffa";
 const mocks = [
+  {
+    request: {
+      query: ListCurrentUserRoomsDocument,
```

Comment:
Also had to mock ListCurrentUserRooms query because i'm refetching queries and it is causing a component rerender and because of that ReactDOM was failing in tests